modules/contracts/adapters/money-market/src/platform_resolver.rs

```diff
@@ -26,9 +26,9 @@ pub(crate) fn resolve_money_market(
     value: &str,
 ) -> Result<Box<dyn MoneyMarketCommand>, MoneyMarketError> {
     match value {
-        #[cfg(feature = "ghost")]
+        #[cfg(feature = "kujira")]
         abstract_kujira_adapter::KUJIRA => {
-            Ok(Box::<abstract_kujira_adapter::dex::Kujira>::default())
+            Ok(Box::<abstract_kujira_adapter::money_market::Kujira>::default())
```

Comment:
noticed we had incorrect feature flag and path for kujira

Reply:
money-market module don't have "ghost" feature, so either feature should be renamed to ghost or vice versa

Reply:
> money-market module don't have "ghost" feature, so either "kujira" feature should be renamed to ghost or vice versa

I'd say we rename to "ghost". Should check with the other adapters as well and ensure we use the protocol name where applicable.